src/main/java/hudson/plugins/jira/JiraMailAddressResolver.java

```diff
@@ -31,7 +31,7 @@ public String findMailAddressFor(User u) {
         String username = u.getId();
 
         for (JiraSite site : JiraGlobalConfiguration.get().getSites()) {
-            JiraSession session = site.getSession();
+            JiraSession session = site.getSession(null);
```

Comment:
Potentially all getSession(null) should be eliminated in this PR, that was one of the things that I was going to fix.